scripts/node/twig_functions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Install Pattern Lab.
+ */
+const shell = require("shelljs");
+const fs = require('fs');
```

Comment:
Because you have `fs-extra` installed, I would use it here to minimize confusion and division